src/tests/ftest/soak/soak_utils.py

```diff
@@ -142,7 +142,7 @@ def run_event_check(self, since, until):
     # when systemctl is enabled add daos events
     events = self.params.get("events", "/run/*")
     # check events on all nodes
-    hosts = list(set(self.hostlist_clients + self.hostlist_servers))
+    hosts = list(set(self.hostlist_servers))
```

Comment:
I believe self.hostliset_servers is already a list.

Reply:
that was a python3 thing that got added to some of the tests...  I can remove it in the next pr for soak  Thanks!